### PR TITLE
New component frequently bought together

### PR DESCRIPTION
**Pull request by haoAddsearch**

### CODE REVIEW

File: src/components/recommendations/recommendations.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  constructor(client, reduxStore, conf, recommendationSettings) {
+    this.client = client;
+    this.conf = conf;
+    this.reduxStore = reduxStore;
+    recommendationSettings.push(conf);
+
+    handlebars.registerHelper('test', (item) => {
+      window.console.log('+++ item', item);
+      return item;
+    });
```
Comment (italo-addsearch) [inline src/components/recommendations/recommendations.js:20]: seems like test code, still needed here?

File: src/components/recommendations/recommendations.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    if (validateContainer(conf.containerId)) {
+      observeStoreByKey(this.reduxStore, 'recommendation', (res) => {
+        if (res.container === conf.containerId) {
+          this.render(res);
```
Comment (italo-addsearch) [inline src/components/recommendations/recommendations.js:25]: shouldn't `render()` be invoked automatically when the state changes? maybe I'm missing something here.

File: src/reducers/configuration.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 const initialState = {};
 
-export default function configuration(state = initialState) {
+export default function configurationzzz(state = initialState) {
```
Comment (italo-addsearch) [inline src/reducers/configuration.js:3]: why `configurationzzz`?

Comment (haoAddsearch) [reply src/reducers/configuration.js]: my mistake, resolved

File: src/components/recommendations/recommendations.js (hunk 2 above)
Comment (haoAddsearch) [reply src/components/recommendations/recommendations.js:20]: That's a design principle for React. In Search UI Library, we define a render method for each component, then observe relating state changes to call the render method.

File: src/components/recommendations/recommendations.js (hunk 1 above)
Comment (haoAddsearch) [reply src/components/recommendations/recommendations.js]: removed.

**Review by italo-addsearch — APPROVED** [review]

**Pull request merged**